packages/react/src/badge/Badge.tsx

```diff
@@ -0,0 +1,58 @@
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Flex } from "../flex";
+import { Text } from "../text";
+import * as styles from "./Badge.css";
+
+const appearances = {
+  danger: { colorScheme: "danger", variant: "accent" },
+  info: { colorScheme: "info", variant: "accent" },
+  neutral: { colorScheme: "neutral", variant: "accent" },
+  primary: { colorScheme: "primary", variant: "subtle" },
+  success: { colorScheme: "success", variant: "subtle" },
+  warning: { colorScheme: "warning", variant: "subtle" },
+} satisfies Record<string, styles.BadgeVariants>;
+
+type BadgeProps = ExtendProps<
+  ComponentPropsWithRef<typeof Flex>,
+  {
+    appearance?: keyof typeof appearances;
+    children: ReactNode;
+  } & styles.BadgeVariants
+>;
+
+export const Badge = forwardRef<HTMLDivElement, BadgeProps>(
+  (
+    {
+      appearance = "neutral",
+      children,
+      className,
+      colorScheme,
```

Comment:
let's default this to `neutral`
```suggestion
      colorScheme = "netural",
```